python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -254,3 +275,35 @@ def _drop_unsupported_aggs(Table values, aggs):
         raise DataError("No numeric types to aggregate")
 
     return result
```

Comment:
@shwina `_drop_unsupported_aggs` may be producing wrong result.
```python

In [1]: D = {'col1': ["sum", "cumsum", "min", "max", "var", "mean", "cumcount"]}

In [2]: result = D.copy()

In [3]: AGGS = {"min", "max", "cumcount"}

In [4]: for i, val in enumerate(D['col1']):
   ...:     if val not in AGGS:
   ...:         print(i, " val=", val)
   ...:         del result['col1'][i]
   ...: 
0  val= sum
3  val= var


In [5]: result
Out[5]: {'col1': ['cumsum', 'min', 'max', 'mean', 'cumcount']}
```

Using a list comprehension should be suitable here.

What are your suggestions?

Reply:
I believe https://github.com/rapidsai/cudf/pull/7818 removes this method entirely. I think at this point it would help to build on top of that PR since there will be conflicts eventually.

Reply:
Merged #7818.